index.bs

```diff
@@ -456,6 +456,10 @@ Insert the following steps in the <a spec="fetch">HTTP-network fetch</a> algorit
     run [=process a fetch storage access for bounce tracking mitigations=]
     given <var ignore>request</var>.
 
+Note: This spec is written for the case where third-party cookies are blocked. Any cookies written by embedded third-party sites are partitioned.
+    Ignore any cookies set in the Set-Cookie header that are ignored by the user agent before committing to storage, such as non-partitioned
+    third-party cookies.
```

Comment:
Avoid writing instructions in a Note, since it's confusing whether they're normative or not in that context. This could be instead:

```suggestion
    Note: If the `Set-Cookie` header includes any cookies that the user agent ignores, for example because it's blocking third-party cookies, they're not considered "stored in the cookie store" in this algorithm.
```